themes/default/content/blog/retainondelete/index.md

```diff
@@ -10,17 +10,17 @@ tags:
 
 ---
 
-Pulumi is frequently used to manage the entire lifecycle of a resource, from creation, to updates, to replacement, to deletion. However, there are some cases where it is important to ensure that a resource's life can extend beyond the lifetime of the Pulumi program that created it. To support these use cases, Pulumi now support a new resource option `RetainOnDelete` which allows a resource to be retained in the backing cloud provider even after it is deleted from the Pulumi stack it is part of.
+Pulumi is frequently used to manage the entire lifecycle of a resource, from creation, to updates, to replacement, to deletion. However, there are some cases where it is important to ensure that a resource's life can extend beyond the lifetime of the Pulumi program that created it. To support these use cases, Pulumi now supports a new resource option `RetainOnDelete` which allows a resource to be retained in a cloud provider even after it is deleted from the Pulumi stack it is part of.
```

Comment:
should it be _the_ instead of _a_ here?

Reply:
This isn't refering to a specific provider so I think a is right here

Reply:
yah just thought it reads kinda like we just pick any cloud provider instead of the cloud provider associated with the resource